### PR TITLE
export validators from addon namespace

### PR DESCRIPTION
**Pull request by luxferresum**
move the validators to the addon namespace an just reflect them in the app namespace for better extensibility.

### CODE REVIEW

File: addon/validators/belongs-to.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2016, Yahoo! Inc.
+ * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
+ */
+
+import Ember from 'ember';
+import Base from 'ember-cp-validations/validators/base';
```
Comment (offirgolan) [inline addon/validators/belongs-to.js]: Can you change all of these to:

```js
import Base from './validators/base';
```

**Comment by luxferresum** [comment]
Do you mean `./base`?

    import Base from './base';

I can do this but I don't think full path is bad.

**Comment by offirgolan** [comment]
Yes

On Monday, March 28, 2016, Lukas Kohler <notifications@github.com> wrote:

> Do you mean ./base?
>
> import Base from './base';
>
> I can do this but I don't think full path is bad.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/offirgolan/ember-cp-validations/pull/152#issuecomment-202641639>
>


-- 
Thanks,
Offir Golan

**Comment by luxferresum** [comment]
Well, I've done that with cff938b

**Comment by offirgolan** [comment]
:+1: Thank you :smile_cat:

**Pull request merged**